src/index.ts

```diff
@@ -0,0 +1,118 @@
+import * as dotenv from "dotenv"
+import { fetchUser } from "./functions/fetch"
+import * as readline from "readline/promises"
+import { searchUsers } from "./functions/show"
+import { userToString } from "./utils"
+
+const handleFetch = async (rl: readline.Interface) => {
+  // Prompt for username
+  const line = await rl.question("Username: ")
+
+  // Parse username
+  const username = line.trim()
+  
+  // Fetch and store user
+  const user = await fetchUser(username)
+
+  // Display result
+  console.log()
+  console.log("===== USER =====")
+  console.log(userToString(user))
+  console.log()
+}
+
+const handleSearch = async (rl: readline.Interface) => {
+  // Prompt for language list
+  const languagesLine = 
+    await rl.question("Languages (example input: C, Python, Typescript): ")
+  
+  // Split list, trim and lowercase it to make it searchable
+  const languages = 
+    languagesLine
+      .split(",")
+      .map(l => 
+        l.trim().toLowerCase()
+      )
+      .filter(l => l)
+
+  
+  // Prompt for location
+  const locationLine = await rl.question("Location: ")
+
+  // Trim location
+  const location = locationLine.trim()
+
+
+  // Search, but only send args if they are defined
+  const users = await searchUsers({
+    languages: languages.length ? languages : undefined,
```

Comment:
Javascript is very _peculiar_ with how it handles `null`, `undefined`, `0` and booleans, which is sometimes a bad thing.

This would be more explicit and safe if the predicate was `languages.length !== 0`

Reply:
That is correct; however, since this is TypeScript, I have the type guarantee that `length` is a number, so doing this comparison should be fine. 
I'll still change it, since it would be the more correct approach in a general sense.